sql/core/src/test/scala/org/apache/spark/sql/CachedTableSuite.scala

```diff
@@ -1212,18 +1212,18 @@ class CachedTableSuite extends QueryTest with SQLTestUtils
   test("SPARK-33290: REFRESH TABLE should invalidate all caches referencing the table") {
     withTable("t") {
       withTempPath { path =>
-        withTempView("tempView1", "tempView2") {
+        withTempView("view1", "view2") {
           Seq((1 -> "a")).toDF("i", "j").write.parquet(path.getCanonicalPath)
           sql(s"CREATE TABLE t USING parquet LOCATION '${path.toURI}'")
-          sql("CREATE VIEW tempView1 AS SELECT * FROM t")
-          sql("CACHE TABLE tempView2 AS SELECT i FROM tempView1")
-          checkAnswer(sql("SELECT * FROM tempView1"), Seq(Row(1, "a")))
-          checkAnswer(sql("SELECT * FROM tempView2"), Seq(Row(1)))
+          sql("CREATE TEMPORARY VIEW view1 AS SELECT * FROM t")
```

Comment:
Or, to be consistent, `tempView1` is better than `tempView1` now since we have `CREATE TEMPORARY VIEW`.

Reply:
The previous Wenchen's comment assumed that `CREATE VIEW`.

Reply:
I see. Will revert the change.